deploy/.env.example

```
@@ -18,6 +18,8 @@ AWS_REGION=ap-northeast-2
 AWS_S3_BUCKET=
 AWS_S3_PUBLIC_BASE_URL=
 AWS_S3_IMAGE_PREFIX=images
+AUTH_RATE_LIMIT_EMAIL_HMAC_SECRET=replace-with-strong-random-secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_RATE_LIMIT_EMAIL_HMAC_SECRET key should go before the AWS_REGION key

(UnorderedKey)